### PR TITLE
MicrodiffAperture inherit from ExporterNState instead of ExporterMotor.

### PR DESCRIPTION
**Pull request by beteva**
Change the MicrodiffAperture to inherit from ExporterNstate rather than ExporterMotor.
I want to ask @all if aperture in/out action should in this class, or treated by a separate class.
Being a separate class is easier as it will be a simple ExporterNState, with the standard set/get_value methods. If we put it here, we would need an extra set/get_in.

### CODE REVIEW

**Comment by marcus-oscarsson** [comment]
I personally think it sounds like a good idea

File: HardwareObjects/MicrodiffAperture.py [hunk 1]
```diff
@@ -17,85 +17,111 @@
...
+from HardwareRepository.HardwareObjects.ExporterNState import ExporterNState
+
+__copyright__ = """ Copyright © 2020 by the MXCuBE collaboration """
+__license__ = "LGPLv3+"
+
+
+class MicrodiffAperture(ExporterNState):
     """MicrodiffAperture class"""
 
+    unit = "mm"
```
Comment (IvarsKarpics) [inline HardwareObjects/MicrodiffAperture.py:47]: Shoud we synchronize units among all hardware objects that are related to beam (slits, aperture, beam). Should it be mm or microns?

Comment (beteva) [reply HardwareObjects/MicrodiffAperture.py:47]: I am not sure it is worth the effort. We have unit and maybe we should use it more often.

File: HardwareObjects/MicrodiffAperture.py [hunk 2]
```diff
@@ -17,85 +17,111 @@
...
+
+Example xml file:
+<object class="MicrodiffAperture">
+  <username>aperture</username>
+  <exporter_address>wid30bmd2s:9001</exporter_address>
+  <value_channel_name>CurrentApertureDiameterIndex</value_channel_name>
+  <state_channel_name>State</state_channel_name>
+  <-- either only factor -->
+  <factor>(0.15, 0.3, 0.63, 0.9, 0.96)</factor>
+  <!-- or complete, corresponding to label: (index, size[mm], factor) -->
```
Comment (IvarsKarpics) [inline HardwareObjects/MicrodiffAperture.py:31]: I assume these are microns?

Comment (beteva) [reply HardwareObjects/MicrodiffAperture.py:31]: You are right, these are microns

Comment (beteva) [reply HardwareObjects/MicrodiffAperture.py]: Yes, repared

File: HardwareObjects/MicrodiffAperture.py (hunk 1 above)
Comment (beteva) [reply HardwareObjects/MicrodiffAperture.py]: Changed unit to microns (um).

**Comment by marcus-oscarsson** [comment]
@IvarsKarpics Are there still changes pending or is this done ?

**Comment by IvarsKarpics** [comment]
We discussed the question regarding the units (to be reviewed later) during the last web meeting. So PR is fine for me.

**Comment by marcus-oscarsson** [comment]
Thanks :)

**Pull request merged**